Gateway/Request/RecurringDataBuilder.php

```diff
@@ -63,9 +63,7 @@ public function build(array $buildSubject): array
         } elseif ($method === PaymentMethods::ADYEN_HPP) {
             $brand = $this->stateData->getPaymentMethodVariant($order->getQuoteId());
             $body = $this->vaultHelper->buildPaymentMethodRecurringData($storeId, $brand);
-        } elseif ($method === PaymentMethods::ADYEN_ONE_CLICK) {
```

Comment:
This should not be removed

Reply:
this change was from #1762, @AlexandrosMor do you mind checking?

Reply:
Fixed in [01af4aa](https://github.com/Adyen/adyen-magento2/pull/1771/commits/01af4aa33bb47ad32aba8593ca70d906c26e700c)